content/search_api_3/latest/searching/offers/name.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+# Name
+
+Using the advanced query parameter `name`, you can search for an exact or partial match with the name of an event or place.
+Likewise you can search for `location.name` to find events that take place on a certain location, or `organizer.name` to search events or places that have a certain organizer.
+
+## Use
+
+|                      | `/offers` | `/events` | `/places` | `/organizers` |
+| -------------------- | --------- | --------- | --------- | ------------- |
+| `name.{lg}`          | x         | x         | x         | x             |
```

Comment:
Not sure if we use `lg` somewhere else? But maybe `language` is better to avoid confusing it with for example `large`?

Reply:
Good point. I'll merge it for now and open a new PR to transform `lg` into `language`